app/src/main/java/com/example/internetshop/presentation/activity/fragments/CategoriesFragment.kt

```diff
@@ -43,30 +43,47 @@ class CategoriesFragment : BaseFragment() {
         super.onViewCreated(view, savedInstanceState)
         val recyclerView = binding?.recyclerViewCategory
         val adapter = CategoryAdapter {
-            CategoriesViewModel.CategoryEvent.OpenCategoryProductListEvent(it.name)
+            viewModel.onClicked(it)
         }
         recyclerView?.let {
-            it.addItemDecoration(VerticalSpaceItemDecoration(32))
+            it.addItemDecoration(VerticalSpaceItemDecoration(52))
```

Comment:
Please add ability to use this item decoration with resources from dimens

Reply:
fixed